src/System.Drawing.Common/tests/mono/System.Drawing/GraphicsTests.cs

```diff
@@ -358,7 +358,22 @@ public void SetSaveReset()
         }
 
         [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Difference in behavior, netfx throws Exception")]
```

Comment:
This one can also use the extension method instead of splitting the test.

Reply:
Yes @safern i'm in the middle of chance...i use WIP on title, is It correct?

Reply:
Yes, that is correct :)